content/en/guides/features/deployment-targets.md

```diff
@@ -7,31 +7,62 @@ category: features
 
 ## Static Hosting
 
-For static hosting (hosting where no server is needed) the target of static needs to be added to your nuxt.config file.
+With static hosting, hosting where no server is needed, you can choose to either host a single page application or a multiple page application, also known as static sties. With static hosting no server is needed, meaning your SPA or static site can be hosted on any serverless hosting or CDN for free. For static hosting the target of `static` needs to be added to your `nuxt.config` file.
 
 ```js{}[nuxt.config.js]
 export default {
   target: 'static' // default is 'server'
 }
 ```
 
-Running nuxt dev with the static target will improve the developer experience:
+### SPA
+
+Single page applications are pages that are rendered only on the client side without the need of a server. To deploy a single page application set the [mode set to `spa`](/guides/features/rendering-modes#spa) and then use the `build` command to build your application.
+
+```js{}[nuxt.config.js]
+export default {
+  target: 'static' // default is 'server'
```

Comment:
Command be `nuxt generate` but i don't think we should set target to either `static` or `server`. SPA with `server` target is also valid so users don't need to specify it.

Reply:
this is where there is a lot of confusion. yes SPA can be either on a static hosting or a server hosting. SPA on netlify or azure static web apps so the deployment target should be static as it is a static deployment (for the sake of logic). So if you don't use target static or server then we need to say for single page applications there is no need to set the target or something similar and for spa you only need to modify the mode. It is as if these 2 pages should almost be one page that shows the 3 options with their corresponding targets and modes. Any suggestions?